workflow/scripts/calculate_background.py

```diff
@@ -0,0 +1,94 @@
+
```

Comment:
This file needs:
- proper logging
- remove comment and add more describing comments
- a unit test

Doesn't necessarily need to be solved in this pull-request, could be added as a new task